tensorboard/plugins/graph/tf_graph_common/op.ts

```diff
@@ -364,10 +471,17 @@ module tf.graph.op {
       'Switch',
       // Ops below are inserted by the compiler.
       '_Arg',
+      '_ArrayToList',
+      '_FusedBatchNormEx',
+      '_ListToArray',
       '_ParallelConcatUpdate',
```

Comment:
When running `SupportedOpsMain` from `tensorflow/compiler/tf2xla/tf2xla_supported_ops.cc`, I'm not seeing these ops
- `_ParallelConcatUpdate`
- `_TPUCompile`
- `_TPUExecute`

in the list of results.  Is keeping them in the allowlist intentional?

Reply:
Yes.  The idea was to support 'legacy' existing graphs.  Although this policy can be changed later.